src/search_algo.hpp

```diff
@@ -196,9 +196,28 @@ prepareScoring(GlobalDataHolder<c_dbIndexType, c_origSbjAlph, c_transAlph, c_red
         seqan::setScoreMatch(context(globalHolder.outfileBlastTab).scoringScheme, options.match);
         seqan::setScoreMismatch(context(globalHolder.outfileBlastTab).scoringScheme, options.misMatch);
 
-        // Seqan3
         globalHolder.scoringScheme.set_simple_scheme(seqan3::match_score{options.match},
                                                      seqan3::mismatch_score{options.misMatch});
+
+        if constexpr (c_redAlph == AlphabetEnum::DNA3BS)
+        {
+            // Seqan2
+            seqan::setScoreBisulfiteMatrix(globalHolder.scoringSchemeAlign, options.match, options.misMatch);
+
+            // Seqan3
+            globalHolder.scoringScheme.set_bisulfite_scheme(seqan3::match_score{options.match},
+                                                            seqan3::mismatch_score{options.misMatch});
+        }
+        else
+        {
+            // Seqan2
+            seqan::setScoreMatch(globalHolder.scoringSchemeAlign, options.match);
+            seqan::setScoreMismatch(globalHolder.scoringSchemeAlign, options.misMatch);
+
+            // Seqan3
+            globalHolder.scoringScheme.set_simple_scheme(seqan3::match_score{options.match},
+                                                         seqan3::mismatch_score{options.misMatch});
```

Comment:
same lines before if-body. Are both required? Where is `globalHolder.scoringScheme` currently used at all?

Reply:
For the scoring in the seed looks promising function? Or not?

Reply:
> For the scoring in the seed looks promising function? Or not?

Ah, yes!

>> same lines before if-body. Are both required?

This is still open.

Reply:
Yes, only the one in the if clause is required, must be an accidental leftover from copy/pasting